File: 0242-valid-anagram/0242-valid-anagram.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isAnagram = function(s, t) {
    //Convert string to array
    const arrayS = s.split("").sort();
    const arrayT = t.split("").sort();
    //If the orderS and orderT is different num of array, it means they are not same, so return false
    if(arrayS.length !== arrayT.length){
        return false;
    }else{
        //If they have same num of array
        //Check orderS and orderT is same or not, approach the each element of array 
        for(let i=0;i<arrayT.length;i++){
            if(arrayS[i]!==arrayT[i]){
                return false;
            }

        }
        return true;

    }

};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str):
                #If the length of sorted strings are not same, 
        #Return False
        if(len(s)!=len(t)):
            return False
        #Sort s and t
        sorted_s="".join(sorted(s))
        sorted_t="".join(sorted(t))

        #Compare each character of strings
        for i in range(len(sorted_s)):
            #If s[i] is not same with t[i], return false
            if(sorted_s[i] != sorted_t[i]):
                return False
        return True #Otherwise, return true
        